src/wallet.ts

```diff
@@ -0,0 +1,34 @@
+import Wallet from './index'
+
+
+const start = async () => {
+    let wallet = new Wallet();
+    await wallet.Aeternity.setup()
+    wallet.Aeternity.networks.addNetwork({
+        name:"Tst",
```

Comment:
Can you add real values in the fields here for adding network instead of random gibberish, please :)

Reply:
Fixed in https://github.com/waellet/waellet-chain-lib/pull/3/commits/fcd33b6848d87b4fdc86668a5d58b8494fb2174b